### ISSUE TITLE
glob pattern needs comma to work!

### ISSUE THREAD

**Issue by eydrian** [cypress-io/cypress#19934]
### Current behavior

having a glob pattern for specs e.g.

```
❯ npx cypress run --spec "projects/cypress/[a-f]*"
```

fails, because it does not contain a comma

whereas 

```
❯ npx cypress run --spec "projects/cypress/[a-f,]*"
```

will successfully run

This was a breaking change in 9.3.0 introduced with https://github.com/cypress-io/cypress/issues/16102

### Desired behavior

```
❯ npx cypress run --spec "projects/cypress/[a-f]*"
```

should work too

### Test code to reproduce

npx cypress run --spec "projects/cypress/[a-f]*"

### Cypress Version

9.3.0

**Comment by tbiethman** [comment]
@eydrian Thanks for reaching out. The code for this is [done](https://github.com/cypress-io/cypress/pull/19789), but this has yet to be released. We'll update this issue and reference the changelog when it's released (9.4.0)

**Comment by tbiethman** [comment]
Please follow https://github.com/cypress-io/cypress/issues/19783 for release updates.
